src/ZendDeveloperTools/Collector/EventCollectorInterface.php

```diff
@@ -23,5 +23,5 @@
      * @param string $id
      * @param Event  $event
      */
-    public function collectEvent($id, Event $event);
+    public function collectEvent($id, EventInterface $event);
```

Comment:
This change can't be implemented until version 2.0.0 of the module, as it is a major BC break